### ISSUE TITLE
Incompatible with project isolation

### ISSUE THREAD

**Issue by ZacSweers** [runningcode/gradle-doctor#180]
When trying out [project isolation](https://gradle.github.io/configuration-cache/), this error is reported

```
Cause 5: org.gradle.api.InvalidUserCodeException: Cannot access project ':services' from project ':'
        at org.gradle.configurationcache.ProblemReportingCrossProjectModelAccess$ProblemReportingProject.onAccess(ProblemReportingCrossProjectModelAccess.kt:970)
        at org.gradle.configurationcache.ProblemReportingCrossProjectModelAccess$ProblemReportingProject.afterEvaluate(ProblemReportingCrossProjectModelAccess.kt:580)
        at com.osacky.doctor.DoctorPlugin$ensureNoCleanTaskDependenciesIfNeeded$1.execute(DoctorPlugin.kt:170)
        at com.osacky.doctor.DoctorPlugin$ensureNoCleanTaskDependenciesIfNeeded$1.execute(DoctorPlugin.kt:31)
        at org.gradle.api.internal.DefaultMutationGuard$2.execute(DefaultMutationGuard.java:44)
        at org.gradle.internal.Actions.with(Actions.java:249)
```

**Comment by runningcode** [comment]
Thanks for reporting. This is due to the clean dependency check. This check will no longer be needed in Gradle 7.4 and will not cause an issue anymore in Gradle 7.4

**Comment by runningcode** [comment]
Fixes in 0.8.0 https://runningcode.github.io/gradle-doctor/changelog/#080

**Comment by ZacSweers** [comment]
FYI this regressed, filed #302
